### PR TITLE
Add "Clear" button to logs

### PR DESCRIPTION
**Pull request by qwerty287**
https://github.com/LycheeOrg/Lychee-front/pull/302

### CODE REVIEW

**Comment by qwerty287** [comment]
CI failure is unrelated.

File: app/Locale/Dutch.php [hunk 1]
```diff
@@ -46,6 +46,7 @@ public function get_locale(): array
 			'DIAGNOSTICS_GET_SIZE' => 'Request space usage',
 			'LOGS' => 'Laat logs zien',
 			'CLEAN_LOGS' => 'Clean Noise',
+			'CLEAR' => 'Leeren',
```
Comment (kamil4) [inline app/Locale/Dutch.php]: My Dutch is a little rusty but I don't believe that's the right word, and the official Dutch dictionary seems to agree (https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/leeren).

**Review by kamil4 — CHANGES_REQUESTED** [review]

File: app/Locale/Dutch.php (hunk 1 above)
Comment (qwerty287) [reply app/Locale/Dutch.php]: Whoops looks like I changed Dutch instead of German ;) ("German" in German is "Deutsch", so I probably misread the filename)

**Review by ildyria — APPROVED** [review]
LGTM. fix conflict & can merge.

**Comment by qwerty287** [comment]
@kamil4 you have to approve because you requested changes and I don't want to use the force-merge option as admin :)

**Review by kamil4 — APPROVED** [review]

**Pull request merged**